src/util/loanSearch/searchStateUtils.js

```diff
@@ -73,3 +74,20 @@ export async function updateSearchState(apollo, loanQueryFilters, allFacets, que
 		}
 	});
 }
+
+/**
+ * Creates a saved search with a name and set of filters
+ * @param {Object} apollo The Apollo client instance
+ * @param {Object} loanQueryFilters The filters for the saved search
+ * @param {string} savedSearchName The name of the saved search
+ * @returns {Promise<Array>} Promise for the results of the mutation
+ */
+export async function createSavedSearch(apollo, loanQueryFilters, savedSearchName) {
```

Comment:
Please create a test for this method in `test/unit/specs/util/loanSearch/searchStateUtils.spec.js`

Reply:
Or create a new test file, based on the other comment I added. Same process though.